src/include/catalog/schema.h

```diff
@@ -4,15 +4,29 @@
 #include <unordered_map>
 #include <utility>
 #include <vector>
+#include "catalog/catalog_defs.h"
 #include "common/constants.h"
 #include "common/macros.h"
 #include "common/strong_typedef.h"
+#include "parser/expression/abstract_expression.h"
 #include "storage/storage_defs.h"
 #include "type/type_id.h"
 #include "type/type_util.h"
 
+namespace terrier {
+class StorageTestUtil;
+}
+
+namespace terrier::tpcc {
+class Schemas;
+}
+
 namespace terrier::catalog {
 
+namespace postgres {
+class Builder;
+}
+
 /**
  * Internal object for representing SQL table schema. Currently minimal until we add more features to the system.
  * TODO(Matt): we should make sure to revisit the fields and their uses as we bring in a catalog to replace some of the
```

Comment:
See TODO. Obsolete now?

Reply:
I think this is obsolete.  The only pending change is properly storing the default values which we have a separate TODO for in the file (line 152).